pkg/storaged/dialog.jsx

```diff
@@ -680,6 +688,28 @@ export const SelectOneRadio = (tag, title, options) => {
     };
 };
 
+export const SelectOneRadioVertical = (tag, title, options) => {
+    return {
+        tag,
+        title,
+        options,
+        initial_value: options.value || options.choices[0].value,
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17226-20231026-132112-61c336d9-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)